src/category_theory/limits/shapes/biproducts.lean

```diff
@@ -322,90 +443,97 @@ instance biprod.snd_epi {X Y : C} [has_binary_biproduct.{v} X Y] :
   split_epi (biprod.snd : X ⊞ Y ⟶ Y) :=
 { section_ := biprod.lift (biprod.inr ≫ biprod.fst) (𝟙 Y) }
 
-@[ext] lemma biprod.hom_ext {X Y Z : C} [has_binary_biproduct.{v} X Y] (f g : Z ⟶ X ⊞ Y)
-  (h₀ : f ≫ biprod.fst = g ≫ biprod.fst) (h₁ : f ≫ biprod.snd = g ≫ biprod.snd) : f = g :=
-binary_fan.is_limit.hom_ext has_binary_biproduct.is_limit h₀ h₁
-
-@[ext] lemma biprod.hom_ext' {X Y Z : C} [has_binary_biproduct.{v} X Y] (f g : X ⊞ Y ⟶ Z)
-  (h₀ : biprod.inl ≫ f = biprod.inl ≫ g) (h₁ : biprod.inr ≫ f = biprod.inr ≫ g) : f = g :=
-binary_cofan.is_colimit.hom_ext has_binary_biproduct.is_colimit h₀ h₁
+@[simp,reassoc]
```

Comment:
I'm slightly confused by this. Isn't there a linter that complains about simp lemmas that can be proved with `by simp`?